universum/modules/output/output.py

```diff
@@ -21,14 +22,15 @@ class Output(Module):
     teamcity_driver_factory = Dependency(TeamcityOutput)
     terminal_driver_factory = Dependency(TerminalBasedOutput)
     html_driver_factory = Dependency(HtmlOutput)
+    github_driver_factory = Dependency(GithubOutput)
 
     @staticmethod
     def define_arguments(argument_parser):
         parser = argument_parser.get_or_create_group("Output", "Log appearance parameters")
-        parser.add_argument("--out-type", "-ot", dest="type", choices=["tc", "term", "jenkins"],
-                            help="Type of output to produce (tc - TeamCity, jenkins - Jenkins, term - terminal). "
-                                 "TeamCity and Jenkins environments are detected automatically "
-                                 "when launched on build agent.")
+        parser.add_argument("--out-type", "-ot", dest="type", choices=["tc", "term", "jenkins", "github"],
+                            help="Type of output to produce (tc - TeamCity, jenkins - Jenkins, term - terminal, "
+                                 "github - Github Actions). TeamCity and Jenkins environments are detected "
+                                 "automatically when launched on build agent.")
```

Comment:
You need to add a comment whether GitHub is detected automatically, or should this output type be always triggered by conscious user action (and if latter, then what action exactly: like, setting up a special environment variable)